evm/v0.5/contracts/dev/PrepaidAggregator.sol

```diff
@@ -41,7 +43,8 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
   }
 
   struct OracleStatus {
-    uint128 withdrawable;
+    uint120 withdrawable;
+    uint8 index;
```

Comment:
Looking ahead, we will soon have an `admin` address which leaves 12 unused bytes. Maybe we just use the extra word now, and the optimization that comes with the additional `SafeMath120` library?

Reply:
We could drop withdrawable down to `uint96` and that leaves room for the 20 byte address.